clvm_tools/curry.py

```diff
@@ -14,10 +14,8 @@
 
 CURRY_OBJ_CODE = assemble(
     """
-    ((c (q ((c 4 (c 2 (c 5 (c 7 (q ()))))))) (c (q ((c (c (q #c) (c (c (q #q) (c 5 (q ())))
-    (c ((c 6 (c 2 (c 11 (q (1)))))) (q ())))) (q ())) (c (i 5 (q (c (q #c) (c (c (q #q)
-    (c 9 (q ()))) (c ((c 6 (c 2 (c 13 (c 11 (q ())))))) (q ()))))) (q 11)) 1))) 1)))
-"""
+((c (q (c 4 (c 2 (c 5 (c 7 (q)))))) (c (q (c (c (q . 5) (c (c (q . 1) 5) (c ((c 6 (c 2 (c 11 (q 1))))) (q)))) (q)) (c (i 5 (q 5 (q . 5) (c (c (q . 1) 9) (c ((c 6 (c 2 (c 13 (c 11 (q)))))) (q)))) (q . 11)) 1)) 1)))
```

Comment:
did anything change here, apart from turning this into a very long line?
also, why did you turn this into a very long line?

Reply:
Yes, quite a bit changed here.
I prefer having this as one long line because it was auto-generated, but I can certainly reformat it.

Reply:
Probably `flake8` complains about the long string, which is why it was on two lines before. I agree it's gross to the point of being nearly useless. Probably better would be a `load_clvm` type function like what's in `chia-blockchain`.